content/api/vision/_index.md

```diff
@@ -25,9 +25,3 @@ The markers in the list have some useful attributes:
 {{% notice tip %}}
 A marker's position can be represented using both the [Cartesian](coordinates/#cartesian-coordinates) and [spherical](coordinates/#spherical-coordinates) coordinate systems.
 {{% /notice %}}
-
-## Marker PDFs
```

Comment:
Why are we removing this? This is a potentially breaking change for people used to coming to this page for these links.

Reply:
It's removed mostly to remove the duplicated content. Perhaps adding a link would've been better? The markers are already a top-level section (theyre in the navbar), so finding the links isn't difficult